addons/account/tests/test_account_move_reconcile.py

```diff
@@ -2357,6 +2567,74 @@ def test_reconcile_rounding_issue(self):
 
         self.assertTrue(invoice.payment_state in ('in_payment', 'paid'))
 
+    def test_reconcile_plan(self):
+        @contextmanager
+        def rollback():
+            savepoint = self.cr.savepoint()
+            yield
+            savepoint.rollback()
```

Comment:
Doesn't `with self.cr.savepoint()` do that?

https://github.com/odoo/odoo/blob/df997b1d01a653dd4272bb5071c7baf5fc1c01e6/odoo/sql_db.py#L82-L96

Reply:
the exit performs a commit, not a rollback